src/components/organisms/ChatSidebar/components/PrivateChats/PrivateChats.tsx

```diff
@@ -17,10 +21,10 @@ import { OnlineUser, PrivateChatPreview, RecipientChat } from "..";
 import "./PrivateChats.scss";
 
 export interface PrivateChatsProps {
-  recipientId?: string;
+  recipient: WithId<ChatUser> | undefined;
```

Comment:
```suggestion
  recipient,: WithId<ChatUser>;
```

Reply:
You meant `recipient?:`?
But that's not the same as `type | undefined` as with `?` you can completely omit the prop but with `type | undefined` you have to pass it but it can be of the `undefined` type.

Reply:
Ah, is it intentianal?